test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsControllerSpec.scala

```diff
@@ -1355,7 +1362,50 @@ class ReliefDetailsControllerSpec
         }
 
         checkIsRedirect(
-          performAction(),
+          performAction(Seq("privateResidentsRelief" -> "1")),
+          routes.ReliefDetailsController.otherReliefsSubmit()
+        )
+      }
+    }
+
+  def residentsReliefSubmittedYesBehaviour(
+    performAction: Seq[(String, String)] => Future[Result],
+    residentsReliefAmount: AmountInPence
+  ): Unit =
+    "redirect to the other reliefs page" when {
+
+      "residents relief is selected as Yes" in {
+        val incompleteAnswers =
+          sample[IncompleteReliefDetailsAnswers].copy(privateResidentsRelief = Some(residentsReliefAmount), None, None)
+        val fillingOutReturn          = sample[FillingOutReturn]
+        val singleDisposalDraftReturn = sample[SingleDisposalDraftReturn]
+        val disposalDate              = sample[DisposalDate]
+        val triageAnswers             = sample[CompleteSingleDisposalTriageAnswers]
+        val taxYear                   = sample[TaxYear]
+
+        val session = sessionWithReliefDetailsAnswers(
+          fillingOutReturn,
+          singleDisposalDraftReturn,
+          Some(incompleteAnswers),
+          None,
+          disposalDate,
+          triageAnswers,
+          taxYear
+        )._1
+        val data = Seq(
```

Comment:
you don't need `data` here - you're doing this test of the display check your answers endpiont which doesn't take any form data. The data necessary for this spec is controlled via what's in session.

Reply:
looks like I've forgotten that while I was in a rush. That was a silly mistake on my side. Done.

Reply:
is `data` still being used somewhere?

Reply:
No, good spot, Deleted.